deploy/operator.yaml

```diff
@@ -429,7 +429,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: dellemc/dell-csm-operator:v0.1.0
+        image: amaas-eos-mw1.cec.lab.emc.com:5036/dell-csm-operator:v0.0.2
```

Comment:
please revert this change. we should not have references to internal repo

Reply:
done